core/src/test/java/com/graphhopper/routing/util/BikeFlagEncoderTest.java

```diff
@@ -290,6 +290,54 @@ public void testSpeedAndPriority() {
         assertEquals(18, encoder.getSpeed(way));
     }
 
+    @Test
+    public void testSmoothness() {
+        ReaderWay way = new ReaderWay(1);
+        way.setTag("highway", "residential");
+        way.setTag("smoothness", "excellent");
+        IntsRef relFlags = encodingManager.createRelationFlags();
+        IntsRef flags = encodingManager.handleWayTags(way, accessMap, relFlags);
+        assertEquals(20, encoder.getAverageSpeedEnc().getDecimal(false, flags), 0.01);
```

Comment:
Can you replace this (and the other asserts) with getSpeedFromFlags?

Reply:
Done in 26f4da7.